waifu2x.ts

```diff
@@ -295,34 +315,38 @@ export default class Waifu2x {
                 fileMap = [...fileMap, ...search]
             }
         }
-        if (!options.limit) options.limit = fileMap.length
-        const retArray: string[] = []
+
+        options.limit = options.limit ?? fileMap.length

```

Comment:
I don't think `0` should be allowed as a valid limit, so keep it the old way or use `||`.

Reply:
About this, what is `limit` for?

Reply:
If you have a directory of 100 images but only want to upscale the first 10 you would pass a `limit` of `10`.